tool/dumper/vkgcPipelineDumper.cpp

```diff
@@ -684,6 +685,7 @@ void PipelineDumper::dumpComputeStateInfo(const ComputePipelineBuildInfo *pipeli
 void PipelineDumper::dumpPipelineOptions(const PipelineOptions *options, std::ostream &dumpFile) {
   dumpFile << "options.includeDisassembly = " << options->includeDisassembly << "\n";
   dumpFile << "options.scalarBlockLayout = " << options->scalarBlockLayout << "\n";
+  dumpFile << "options.resourceLayoutScheme = " << options->resourceLayoutScheme << "\n";
```

Comment:
I think this file should also include the resource layout scheme in the hash.

Reply:
Oh, I think hash includes resource mapping, so I didn't add it.  After checking again, it only is added when relocatable is false. Thanks, I will add it.